src/mochiweb_headers.erl

```diff
@@ -215,6 +215,9 @@ tokenize_header_value(undefined) ->
 tokenize_header_value(V) ->
     reversed_tokens(trim_and_reverse(V, false), [], []).
 
+trim_leading_and_trailing_ws(S) ->
```

Comment:
Since otp20 there is string:trim(S, both) that would help you a long way...
But possibly some other guarantees are given here.

Reply:
I think there may be a difference in the handling of "\r", which isn't referred to in this function.  For now, I'm preferring to de-risk the change and not make any assumptions.